marketplace/bundles/plan-marshall/skills/workflow-integration-git/scripts/git-workflow.py

```
@@ -528,25 +528,38 @@ def cmd_analyze_diff(args):
+    ``--directory`` is load-bearing twice over. It collapses a fully-ignored
+    directory to a single trailing-slash entry, which is the form
+    :func:`_split_ignored` partitions on and :func:`_is_ignored` prefix-matches
+    — without it that arm receives no input and can never fire for any path. It
+    also keeps the query small enough to finish: measured on this repository the
+    flagged query returns 207 entries against 213256 unflagged, and the
+    unflagged form routinely exceeded the timeout below.
     """
     try:
         result = subprocess.run(
-            ['git', 'ls-files', '--others', '--ignored', '--exclude-standard'],
+            ['git', 'ls-files', '--others', '--ignored', '--exclude-standard', '--directory'],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)